README.md

```diff
@@ -0,0 +1,46 @@
+This API allows simplified access to Google Sheets.
```

Comment:
The styleguides contain a README template:

https://github.com/alphagov/styleguides/blob/master/use-of-READMEs.md

Not everything applies, but the basic structure can be used (see https://github.com/alphagov/govuk_schemas for example).